README.md

```diff
@@ -275,10 +264,25 @@ Setter for value that will fire all appropriate handlers/tests. Can also be done
 
 Passing `true` as second argument will skip validation. This is mainly for internal use.
 
+#### Setting input.value on non-user input
+This module assumes that the value of the input element will be set by the user.  This is the only event that can be reliably listened for on an input element.  If you have a third-party library (i.e. Bootstrap or jQuery) that is going to be affecting the input value directly you will need to let your model know about the change via `setValue`.
+
+```javascript
+var myInput = new InputView({
+    name: 'date'
+});
+myInput.render();
+document.body.appendChild(myInput.el);
+
+$('[name=address]').datepicker({
```

Comment:
:+1: for the targeted example :).  I think there's some syntax stuff here (missing `}` ?).  Alas, the point is made